kernel/src/lib.rs

```diff
@@ -121,11 +124,16 @@ fn print_flower() -> Result<(), terminal::TerminalOutputError<()>> {
     const FLOWER_STEM: &'static str = include_str!("resources/art/flower_stem.txt");
 
     let mut stdout = terminal::STDOUT.write();
-    let old = stdout.cursor_pos();
+    let old = stdout.cursor_pos().expect("Terminal must support cursor");
 
     stdout.write_string_colored(FLOWER, color!(LightBlue on Black))?;
     stdout.write_string_colored(FLOWER_STEM, color!(Green on Black))?;
-    stdout.set_cursor_pos(old)
+    stdout.set_cursor_pos(old)?;
+
+    write!(serial::PORT_1.lock(), "{}", FLOWER).unwrap();
```

Comment:
Terribly minor, maybe we should make a macro specifically to output to serial 1?

Reply:
Phil opp does this and it should be relatively simple, so why not?